doc/getting-started.rst

```diff
@@ -17,6 +17,35 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
 
 See :doc:`features/parsing` for additional ways to instantiate a Money object from strings.
 
+Accepted integer values
+-----------------------
```

Comment:
I still think this does not belong to a Getting started guide, but let's not block the PR with it.